bin/calcdeps.js

```diff
@@ -377,7 +377,7 @@ deps.build = function(infos) {
 deps.resolve = function(info, hash, ordered, seen) {
   info.requires.forEach(function(ns) {
     if (!(ns in hash)) {
-      $.util.error($.util.format(
+      console.error($.util.format(
```

Comment:
revert this file as well

Reply:
util.error was deprecated (https://nodejs.org/docs/latest-v0.12.x/api/util.html#util_util_error) in favor of console.error.

in 0.10, console.error was already there (https://nodejs.org/docs/latest-v0.10.x/api/console.html#console_console_error_data), so updated to be compatible with all versions of node.